docs/05/b.md

```diff
@@ -103,7 +103,7 @@ highlights what a monad is.
 4. `Pole(1, 4).some >>= {_.landLeft(-1)}` happens, resulting to `none[Pole]`. The difference is too great, and pole becomes off balance.
 5. `none[Pole] >>= {_.landRight(-2)}` results automatically to `none[Pole]`.
 
-In this chain of monadic functions, the effect from function is carried over to another.
+In this chain of monadic functions, the effect from one function is carried over to the next.
```

Comment:
Alternatively "from *a* function is carried over to another."
I was trying for a strong sense of causality.